wiktextract/extractor/zh/page.py

```diff
@@ -323,17 +323,30 @@ def split_pronunciation_tags(text: str) -> List[str]:
 
 def extract_pronunciation_item(
     wxr: WiktextractContext, lang_code: str, node: WikiNode, tags: List[str]
-) -> Union[Dict, str]:
+) -> Optional[Union[Dict, str]]:
     expanded_text = clean_node(wxr, None, node.children)
-    # breakpoint()
     if expanded_text.startswith("File:"):
         return expanded_text
     else:
-        sound_tags, ipa = re.split("：|:", expanded_text, 1)
-        data = {"tags": list(set(tags + split_pronunciation_tags(sound_tags)))}
-        ipa_key = "zh-pron" if lang_code == "zh" else "ipa"
-        data[ipa_key] = ipa.strip()
-        return data
+        sound_tags, *ipa = re.split("：|:", expanded_text, 1)
```

Comment:
Using an r""-string is probably best for when someone needs to edit this to add something that has been escaped and they don't realize the string isn't an r-string.

Reply:
Thanks, I somehow forget to use the raw string for regex pattern at here.